Exiled.API/Features/Player.cs

```diff
@@ -79,27 +79,20 @@ public class Player
 #pragma warning restore SA1401
 
         private readonly IReadOnlyCollection<Item> readOnlyItems;
+        private readonly HashSet<EActor> components = new();
 
-        /// <summary>
-        /// The running speed of the player.
-        /// </summary>
         private float? runningSpeed;
-
-        /// <summary>
-        /// The walk speed of the player.
-        /// </summary>
         private float? walkingSpeed;
 
         private ReferenceHub referenceHub;
         private CustomHealthStat healthStat;
         private Role role;
-        private HashSet<EActor> components = new();
 
         /// <summary>
         /// Initializes a new instance of the <see cref="Player"/> class.
         /// </summary>
         /// <param name="referenceHub">The <see cref="global::ReferenceHub"/> of the player to be encapsulated.</param>
-        public Player(ReferenceHub referenceHub)
+        internal Player(ReferenceHub referenceHub)
```

Comment:
no

Reply:
Wtf why no ? use Player.Get()

Reply:
???

Reply:
That ctor should be non-public, cuz it can broke Player.List

Reply:
> That ctor should be non-public, cuz it can broke Player.List

Exiled.API.Publicized ;)